doc/whats_new/v1.1.rst

```diff
@@ -923,6 +923,11 @@ Changelog
 :mod:`sklearn.neighbors`
 ........................
 
+- |Feature| :class:`neighbors.KernelDensity` bandwidth parameter now accepts
+  definition using Scott's and Silvermann's estimation methods.
+  :pr:`10468` by :user:`Ruben <icfly2>` and :pr:`22993` by
+  :user:`Jovan Stojanovic <jovan-stojanovic>`.
```

Comment:
Could you move this entry in `doc/whats_new/v1.2.rst`.
The file should already exist. You might need to create the section thought.